LondonBritishLibrary/orient/BLorient9036.xml

```diff
@@ -0,0 +1,297 @@
+                                    <locus target="#111r"/>
+                                    <title ref="LIT1828Mahale#Mary"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.1.14">
+                                    <locus from="111r" to="111v"/>
+                                    <title ref="LIT1828Mahale#Zachariah"/>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.1.15">
+                                    <locus target="#111v"/>
+                                    <title ref="LIT1828Mahale#Simeon"/>
+                                </msItem>
+                            </msItem>
+                            <msItem xml:id="ms_i1.2.2">
+                                <locus from="111v" to="118r"/>
+                                <title ref="LIT2362Songof"/>
+                                <note>The text is divided into five sections.</note>
+                                <msItem xml:id="ms_i1.2.2.1">
+                                    <title>First section</title>
+                                    <note>Title missing</note>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.2.2">
+                                    <title>Second section</title>
+                                    <incipit xml:lang="gez">ካልዕ፡ ማኅልይ፡ ዘይዜኑ፡ ልደቶ፡ ለዋህድ።</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.2.3">
+                                    <title>Third section</title>
+                                    <incipit xml:lang="gez">ሳልስ፡ መኀልይ፡ ዘይነግር፡ ዕርገተ፡ ትስብእት፡ ምስለ፡ መላእክት፡ ድኅረ፡ ፩ከዊን።</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.2.4">
+                                    <title>Fourth section</title>
+                                    <incipit xml:lang="gez">ራብዕ፡ ዘይነግር፡ ውዳሴ፡ መርዓት፡ ለመርዓዊሃ።</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i1.2.2.5">
+                                    <title>Fifth section</title>
+                                    <incipit xml:lang="gez">እምዚአሃ፡ ዘዚአሃ፡ ለቤተ፡ ክርስቲያን።</incipit>
+                                </msItem>
+                            </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i1.3">
+                            <locus from="118va" to="130ra"/>
+                            <title ref="LIT2509Weddas"/>
+                            <note>Arranged by the days of the week, with Monday's lesson first.</note>
+                            <incipit xml:lang="gez">ወሶበ፡ ባረከቶ፡ እግዝ፡ እትነ፡ ድንግል፡ ለኤፍሬም፡ ከሠተ፡ አፋሁ፡ ወይቤ፡ ውዳሴሃ፡ ለእግዝእትነ፡ ማርያም፡ ድንግል፡ ወላዲተ፡ 
+                                አምላክ፡ ዘይት<supplied reason="undefined" resp="PRS8999Strelcyn">ነበብ፡</supplied><title> በዕለተ፡ ሰኑይ፡</title></incipit>
```

Comment:
also here one could have ms_i1.3.1 `LIT2509Weddas#Monday`, ms_i.2.3 `LIT2509Weddas#Tuesday` etc

Reply:
Done, also for BLorient4611, BLorient13266 and BLorient12114A.